app/src/main/res/values/strings.xml

```diff
@@ -98,31 +98,31 @@
     <string name="restore_back">Don\'t restore</string>
     <string name="restore_invalid_location_title">No backups found</string>
     <string name="restore_invalid_location_message">We could not find any backups at this location.\n\nPlease choose another location that contains a %s folder.</string>
-    <string name="restore_set_error">An error occurred while loading the backups.</string>
-    <string name="restore_set_empty_result">No suitable backups found at given location.\n\nThis is most likely due to a wrong recovery code or a storage error.</string>
+    <string name="restore_set_error">Could not load the backups.</string>
+    <string name="restore_set_empty_result">No suitable backups found there.\n\nThis is most likely due to a wrong recovery code or a storage error.</string>
     <string name="restore_installing_packages">Re-installing Apps</string>
     <string name="restore_next">Next</string>
-    <string name="restore_restoring">Restoring Backup</string>
-    <string name="restore_magic_package">System Package Manager</string>
+    <string name="restore_restoring">Restoring backup…</string>
+    <string name="restore_magic_package">System package manager</string>
     <string name="restore_app_no_data">App reported no data for backup</string>
     <string name="restore_app_not_allowed">App doesn\'t allow backup</string>
     <string name="restore_app_not_installed">App not installed</string>
     <string name="restore_app_quota_exceeded">Backup quota exceeded</string>
-    <string name="restore_finished_success">Restore complete</string>
-    <string name="restore_finished_error">An error occurred while restoring the backup.</string>
+    <string name="restore_finished_success">Restored</string>
+    <string name="restore_finished_error">Could not restore the backup.</string>
     <string name="restore_finished_button">Finish</string>
     <string name="storage_internal_warning_title">Warning</string>
-    <string name="storage_internal_warning_message">You have chosen internal storage for your backup. This will not be available when your phone is lost or broken.</string>
-    <string name="storage_internal_warning_choose_other">Choose Other</string>
+    <string name="storage_internal_warning_message">You have chosen internal storage for your backup, and it will not help you if your phone is lost or broken.</string>
```

Comment:
```suggestion
    <string name="notification_error_text">Could not perform backup.</string>
```
"Backup failed to run." is quirky English for no good reason.
```suggestion
    <string name="storage_internal_warning_message">You have chosen internal storage for your backup, and it will not help you if your device is lost or broken.</string>
```